### ISSUE TITLE
Please add existing Auction Date to Foundations API

### ISSUE THREAD

**Issue by quigger** [reapit/foundations#7947]
Related to: https://github.com/reapit/foundations/issues/7930

There is an Auction Date field present in the Reapit system - can this be utilised within the Foundations feed to Homeflow.

**Comment by quigger** [comment]
@plittlewood-rpt I do not think this time period is going to be acceptable to the GTH client. I thought the Auction Date field was a core Reapit value? Should it not be part of Foundations?

**Comment by quigger** [comment]
Hi @plittlewood-rpt 

OK, perhaps you need to inform Ankit and team in support.

Ankit said:
"Jason, there is an Auction Date field present in the system that could be utilised within the Foundations feed to Homeflow"

And told us to request the data via Foundations?

**Comment by plittlewood-rpt** [comment]
I will feed this back - the way that's being worded suggests it's already available rather than asking you to make a feature request. Thank you

**Comment by quigger** [comment]
Thanks @plittlewood-rpt 

Yes, we based our dev and project planning on the fact that it sounded like it already existed in Reapit Cloud, and so would be already available in Foundations.

**Comment by plittlewood-rpt** [comment]
I would recommend always checking over the Swagger documentation to confirm if something is already available and using this information when looking at your own timelines.

**Comment by quigger** [comment]
@plittlewood-rpt OK, but you'd hope that Reapit would inform correctly when making suggestions in tickets.

**Comment by plittlewood-rpt** [comment]
As mentioned above I will (and have) fed this back to our service desk team manager
